### DISCUSSION TITLE
Central two thirds of the screen

### DISCUSSION THREAD

**Discussion by XaviCliment** [rxhanson/Rectangle#406] [Ideas]
Hi! I have a 34" ultrawide monitor and lots of times I need to resize the windows to fill the central two thirds of the screen (it's one of the most comfortable sizes for this kind of monitor). Could you add this feature?

Thanks in advance!

**Comment by rxhanson** [comment]
Are you looking for something different from the existing "Center Third" action?

**Reply by XaviCliment** [reply]
Oh I’m sorry, I made a mistake. I was talking about the central two thirds of the screen because this option is not implemented yet. Could you add it? (I’ve already edit the tittle and message of the issue so that people don’t get confused)

Thank you!!

**Comment by dmland** [comment]
I know it's not as convenient as a single command, but "First Two Thirds" (or Last), followed by "Center" achieves this.

**Reply by XaviCliment** [reply]
Oh, I didn’t know that! In this case, I’ll do it like this, I don’t care if I have to use both actions 😁 Thanks a lot for your reply!

**Comment by rmseng** [comment]
If at all possible, I'd also love to have this feature added.  It'd be particularly useful for centering a window with vertical content that isn't already flowed in a column format, such as Wikipedia opened in a browser.

By the way - a personal thanks to you for creating and maintaining this software.  I've been a Mac user for years, and love it, but the one thing it's always been missing is sane window management.  This OSS tool is a blessing.

**Comment by rxhanson** [comment]
If you don't currently use "Almost Maximize", you can modify it to be center two thirds with the commands listed in the readme here: 
https://github.com/rxhanson/Rectangle#almost-maximize

Just set height to 1 and width to something like 0.6666.

There's also this command if you prefer cycling thirds for the center half command:
https://github.com/rxhanson/Rectangle/blob/master/README.md#cycle-thirds-on-repeated-center-half-commands

**Reply by XaviCliment** [reply]
Oh that’s perfect! I’ll try it out! Thanks for this great app and for all your help ☺️

**Reply by rmseng** [reply]
Ah, thank you!  The 'cycle thirds on repeated center half' will work great for me.
